.NET/Microsoft.Recognizers.Text.Number/French/Extractors/CardinalExtractor.cs

```diff
@@ -8,24 +8,8 @@ namespace Microsoft.Recognizers.Text.Number.French
 {
     public class CardinalExtractor : BaseNumberExtractor
     {
-        internal sealed override ImmutableDictionary<Regex, TypeTag> Regexes { get; }
-
-        protected sealed override string ExtractType { get; } = Constants.SYS_NUM_CARDINAL;
-
-        private static readonly ConcurrentDictionary<string, CardinalExtractor> Instances = 
-            new ConcurrentDictionary<string, CardinalExtractor>();
-
-        public static CardinalExtractor GetInstance(string placeholder = NumbersDefinitions.PlaceHolderDefault)
-        {
-
-            if (!Instances.ContainsKey(placeholder))
-            {
-                var instance = new CardinalExtractor(placeholder);
-                Instances.TryAdd(placeholder, instance);
-            }
-
-            return Instances[placeholder];
-        }
+        private static readonly ConcurrentDictionary<string, CardinalExtractor> Instances =
+         new ConcurrentDictionary<string, CardinalExtractor>();

```

Comment:
Maybe replace this "whitespace" with "tab"?

Reply:
@guom08 updated with full 4-whitespace (a "tab") indentation. It had a single whitespace before.